code/modules/clothing/suits/armor.dm

```diff
@@ -74,6 +74,90 @@
 		usr.update_action_buttons()
 		return 1
 
+/obj/item/clothing/suit/armor/vest/breach1
+	name = "chemical corps vest"
+	desc = "A reinforced combat vest from solfed surplus stockpiles. Originally an enviroment protection suit clad in armor plating for full body protection and points for mounting specialist equipment. This set seems void of typical additions."
+	icon_state = "breach1"
+	item_state = "breach1"
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
+	stiffness = LIGHT_STIFFNESS
+	price_tag = 600
+	armor_list = list(
+		melee = 10,
+		bullet = 11,
+		energy = 10,
+		bomb = 25,
+		bio = 100,
+		rad = 70
+	)
+
+/obj/item/clothing/suit/armor/vest/breach1/verb/toggle_style()
+	set name = "Adjust Style"
+	set category = "Object"
+	set src in usr
+
+	if(!isliving(loc))
+		return
+
+	var/mob/M = usr
+	var/list/options = list()
+	options["Base"] = "breach1"
+	options["Brown"] = "breach1_brown"
+
+	var/choice = input(M,"What kind of style do you want?","Adjust Style") as null|anything in options
+
+	if(src && choice && !M.incapacitated() && Adjacent(M))
+		icon_state = options[choice]
+		to_chat(M, "You adjusted your attire's style into [choice] mode.")
+		update_icon()
+		update_wear_icon()
+		usr.update_action_buttons()
+		return 1
+
+/obj/item/clothing/suit/storage/vest/breach2
+	name = "chemical breacher vest"
+	desc = "A reinforced combat vest from solfed surplus stockpiles. Originally an enviroment protection suit clad in armor plating for full body protection and points for mounting specialist equipment. This comes with enough pouches to hold any mission essential equipment like pens, pencils, chocolate or a brick of plastique."
+	icon_state = "breach2"
+	item_state = "breach2"
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
+	stiffness = LIGHT_STIFFNESS
+	price_tag = 1100
```

Comment:
See above, it's a direct upgrade over the other one and while it's more expensive it aint that much more expensive and has no downsides to account for the benefit.

Reply:
This was my request! Its so if you buy and craft it, you can turn a proofit - sweetshop CT labour :3c

Reply:
Profit isn't really an issue but the fact you can make it relatively easily for a massive benefit and no downside is....not ideal.